server/src/services/UserServices.js

```diff
@@ -14,13 +14,12 @@ const UserServices = {
    * @param {String} name
    * @return {Promise<void>}
    */
-  async validateUser(email, password, name) {
+  async validateUser(email, password, firstName, lastName) {
     const user = await UserModel.findOne({ email });
-    console.log(user);
     if (user) {
       throw new Error('DUPLICATE_EMAIL');
     }
-    if (name === '' || name === undefined || name === null) {
+    if (firstName === '' || firstName === undefined || firstName === null || lastName === '' || lastName === undefined || lastName === null ) {
       throw new Error('MISSING_NAME');
     }
```

Comment:
In general, I'd prefer to not be doing this in our validation function as this will be caught as an error already when you try and create an entry in mongo (if it is set up to not accept empty strings for these properties). Essentially, this just becomes another piece of code we have to maintain whenever we make changes to the schema of our data.
```suggestion

```